src/main/java/pl/javastart/task/NumberGuesser.java

```diff
@@ -0,0 +1,53 @@
+package pl.javastart.task;
+
+import java.util.Scanner;
+
+class NumberGuesser {
+    private int usersNumber;
```

Comment:
To chyba nie musi być polem, może być zmienną lokalną: i tak przekazujesz wszędzie jako parametr

Reply:
Racja. To była pozostałość po moim początkowym pomyśle na rozwiązanie tego zadania - ustaliłem wówczas tę zmienną jako pole, by uniknąć wielokrotnego ponownego definiowania jej w innych częściach programu. Ostatecznie jednak strategię podejścia do rozwiązania zmieniłem, a o zastąpieniu tego pola zmienną lokalną nie pomyślałem.